web/docs/database/entities.md

```diff
@@ -1,3 +1,66 @@
+
+- [Basic intro and examples](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema)
+- [A more exhaustive language specification](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference)
+
+## Defining an Entity
+
+As mentioned, an `entity` declaration represents a database model.
+
+Each `Entity` declaration corresponds 1-to-1 to [Prisma's data model](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model). Here's how you could define an entity that represents a Task:
+
+```wasp
+entity Task {=psl
+    id          Int     @id @default(autoincrement())
+    description String
+    isDone      Boolean @default(false)
+psl=}
+```
+
+Let's go through this declaration in detail:
+
+- `entity Task` - This tells Wasp that we wish to define an Entity (i.e., database model) called `Task`. Wasp automatically creates a table called `tasks`.
+- `{=psl ... psl=}` - Wasp treats everything that comes between the two `psl` tags as [PSL](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema).
+
+The above PSL definition tells Wasp to create a table for storing Tasks where each task has three fields (i.e., the `tasks` table has three columns):
+
+- `id` - An integer value serving as a primary key (i.e., ID). The database automatically generates it by incrementing the previously generated `id`.
+- `description` - A string value for storing the task's description.
+- `isDone` - A boolean value indicating the task's completion status. If you don't set it when creating a new task, the database sets it to `false` by default.
+
+### Workflow with Entities
+
+Let's see how you can define and work with Wasp Entities:
+
+1. Create/update some Entities in your `.wasp` file.
+2. Run `wasp db migrate-dev`. This command syncs the database model with the Entity definitions in your `.wasp` file. It does this by creating migration scripts.
+3. Migration scripts are automatically placed in the `migrations/` folder. Make sure to commit this folder into version control.
+4. Use Wasp's JavasScript API to work with the database when implementing Operations (we'll cover this in detail on the [next page](/docs/database/operations.md))
+
+#### Using Entities in Operations
+
+Most of the time, you will be working with Entities within the context of [Operations (Queries & Actions)](/docs/database/operations.md). We'll see how that's done on the next page.
+
+#### Using Entities directly
+
+If you need more control, you can directly interact with Entities by importing and using the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client/crud). We recommend sticking with conventional Wasp-provided mechanisms, only resorting to directly using the Prisma client only if you need a feature Wasp doesn't provide.
+
+You can only use the Prisma Client in your Wasp server code. You can import it like this:
+
+```js
```

Comment:
I would put this in a JS/TS switch (just to keep it consistent) and also maybe expand the snippet to include usage e.g. `prismaClient.tasks.create({ ... })`